src/net_processing.cpp

```diff
@@ -271,7 +271,7 @@ class PeerManagerImpl final : public PeerManager
 {
 public:
     PeerManagerImpl(const CChainParams& chainparams, CConnman& connman, CAddrMan& addrman,
-                    BanMan* banman, CScheduler& scheduler, ChainstateManager& chainman,
+                    BanMan* banman, ChainstateManager& chainman,
                     CTxMemPool& pool, bool ignore_incoming_txs);
```

Comment:
Nit: `CTxMemPool& pool,` can be moved to the line above to fill in the extra space that has been created.

There are four other lines in the commit that could use the same spacing fix:
net_processing.cpp:1401
net_processing.cpp:1408
net_processing.h:41
test/util/setup_common.cpp:201

Reply:
Thank you, I thought about doing that but decided it would be better to minimize the diff. Also, `clang-format-diff.py` didn't suggest doing that.